ocs_ci/ocs/ui/validation_ui.py

```diff
@@ -565,3 +566,18 @@ def check_capacity_breakdown(self, project_name, pod_name):
             logger.error(f"The pod {pod_name} not found on capacity_breakdown")
             res = False
         return res
+
+    def validate_unprivileged_access(self):
+        """
+        Function to verify the unprivileged users can't access ODF dashbaord
+        """
+        self.do_click(self.validation_loc["developer_dropdown"])
+        self.do_click(self.validation_loc["select_administrator"], timeout=5)
+        try:
+            self.navigate_odf_overview_page()
+        except TimeoutException as ex:
```

Comment:
Pls create a custom exception for broader understanding

Reply:
I feel custom exception is unnecessary if its not reusable, thats why `logger.info`